packages/drivers/driver-base/src/documentDeltaConnection.ts

```diff
@@ -736,13 +736,13 @@ export class DocumentDeltaConnection
 	}
 
 	private createErrorObjectWithProps(
-		handler: string,
+		socketEvent: string,
 		error?: any,
 		props?: ITelemetryBaseProperties,
 		canRetry = true,
 	): IAnyDriverError {
```

Comment:
I think you should also add the new prop where this function is called in props argument, so that it gets added to error here too.